src/gracetabnotegroup.js

```diff
@@ -0,0 +1,147 @@
+// [VexFlow](http://vexflow.com) - Copyright (c) Mohit Muthanna 2010.
+// @author Balazs Forian-Szabo
+//
+// ## Description
+//
+// This file implements a container for grace
+// notes to be rendered on a tab stave.
+//
+// See `tests/gracetabnote_tests.js` for usage examples.
+
+
+import { Vex } from './vex';
+import { Flow } from './tables';
+import { Modifier } from './modifier';
+import { Formatter } from './formatter';
+import { Voice } from './voice';
+import { TabTie } from './tabtie';
+
+// To enable logging for this class. Set `Vex.Flow.GraceNoteGroup.DEBUG` to `true`.
+function L(...args) { if (GraceTabNoteGroup.DEBUG) Vex.L('Vex.Flow.GraceTabNoteGroup', args); }
+
+export class GraceTabNoteGroup extends Modifier {
```

Comment:
Do we really need this class? Can't we just use `GraceNoteGroup` instead? (I'd like to minimize duplicated code and logic as much as possible.)